subprojects/core/src/integTest/groovy/org/gradle/internal/operations/logging/LoggingBuildOperationProgressIntegTest.groovy

```diff
@@ -138,6 +148,16 @@ class LoggingBuildOperationProgressIntegTest extends AbstractIntegrationSpec {
         operations.parentsOf(downloadEvent).find {
             it.hasDetailsOfType(ExecuteTaskBuildOperationType.Details) && it.details.taskPath == ":resolve"
         }
+
+        def runBuildProgress = operations.only('Run build').progress
+        runBuildProgress.size() > 0
+        runBuildProgress[0].details.logLevel == 'QUIET'
```

Comment:
This looks like it's failing on CI? https://e.grdev.net/s/i6zyrydi67tec/tests/moemwyc2h3row-7kglck7pkegeg

Reply:
interesting. I'll have a look

Reply:
@big-guy turns out that was a valid failure. Its passing now on CI but when I run ready-to-merge I see other unrelated (very likely just flaky) tests going red :/